db/migrate/20240303120759_add_proposed_expiry_date_to_validation_requests.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddProposedExpiryDateToValidationRequests < ActiveRecord::Migration[7.1]
+  def change
+    add_column :validation_requests, :proposed_expiry_date, :datetime
```

Comment:
Just checking: will we be using the time part of this, or will it only ever be a date? If the latter it might be better to use a `date` column, because we've had cases in the past where we need to round a datetime to the beginning or the end of the day every time it's referenced.

Reply:
Yes makes much more sense!